src/runner.rs

```diff
@@ -139,9 +144,57 @@ fn parse_test(test_path: &Path, config: &TestConfig) -> InnerTestResult<Test> {
         expected_stdout,
         expected_stderr,
         expected_exit_status,
+        rest,
     })
 }
 
+fn overwrite_test(test_path: &PathBuf, config: &TestConfig, output: &Output, test: &Test) -> std::io::Result<()> {
+    // Maybe copy the file so we don't remove it if we fail here?
+    let mut file = File::create(test_path)?;
+
+    file.write_all(test.rest.as_bytes())?;
+
+    if !test.command_line_args.is_empty() {
+        writeln!(file, "{}{}", config.test_args_prefix, test.command_line_args)?;
+    }
+
+    if Some(0) != output.status.code() {
+        writeln!(
+            file,
+            "{} {}",
+            config.test_exit_status_prefix,
+            output.status.code().unwrap_or(0)
+        )?;
+    }
+
+    // Doesn't handle \r correctly!
+    if output.stdout.len() != 0 {
+        writeln!(file, "{}", config.test_stdout_prefix)?;
+        for line in output.stdout.split(|c| *c == 0x0A) {
```

Comment:
Lets avoid hex literals, it is easier to see what we're looking for if we test for `'\n'` instead

Reply:
Absolutely!